packages/drift_sqlite_async/test/generated/database.g.dart

```diff
@@ -0,0 +1,189 @@
+// GENERATED CODE - DO NOT MODIFY BY HAND
```

Comment:
Should all of this generated code be included?

Reply:
The original `drift_sqlite_async` database has the file included and it makes running the tests easier.